### ISSUE TITLE
today's git: lutris does not launch

### ISSUE THREAD

**Issue by legluondunet** [lutris/lutris#3435]
Hey, I just made a git pull and Lutris does not start anymore:

```
$ ./lutris 
Traceback (most recent call last):
  File "./lutris", line 52, in <module>
    from lutris.gui.application import Application  # pylint: disable=no-name-in-module
  File "/home/romain/Autres_applications/Jeux/Lutris/lutris/lutris/gui/application.py", line 33, in <module>
    from lutris import settings
  File "/home/romain/Autres_applications/Jeux/Lutris/lutris/lutris/settings.py", line 36, in <module>
    raise ValueError(sys.argv)
ValueError: ['./lutris']
```

**Comment by Nicceboy** [comment]
I have similar issue. However, running as root it works for some reason.

**Comment by aTosser** [comment]
fixed by commit 4497ef4
